eZ/Bundle/PlatformBehatBundle/Helper/Xpath.php

```diff
@@ -0,0 +1,259 @@
+<?php
+/**
+ * File containing the Xpath helper for PlatformBehatBundle.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+namespace EzSystems\PlatformBehatBundle\Helper;
+
+use Behat\Mink\Session;
+
+/**
+ * This class eases the xpath creation and handling also has methods to easy search for certain content.
+ */
+class Xpath
+{
+    /**
+     * Initialize class.
+     *
+     * @param \Behat\Mink\Session $session Behat session
+     */
+    public function __construct(Session $session)
+    {
+        $this->session = $session;
+    }
+
+    /**
+     * This is a simple shortcut for
+     * $this->session->getPage()->getSelectorsHandler()->xpathLiteral().
+     *
+     * @param string $text
+     */
+    public function literal($text)
+    {
+        return $this->session->getSelectorsHandler()->xpathLiteral($text);
+    }
+
+    /**
+     * Find all elements that match XPath.
+     *
+     * @param string $xpath XPath to find the elements
+     *
+     * @return \Behat\Mink\Element\NodeElement[] Array with NodeEelments that match
+     */
+    public function findXpath($xpath)
+    {
+        return $this->session->getPage()->findAll('xpath', $xpath);
+    }
+
+    /**
+     * Make XPath for a specific element/object using Behat selectors.
+     *
+     * @param string $element Type of element for the XPath
+     * @param string $search String to search
+     *
+     * @return string XPath for the element/object
+     */
+    public function makeElementXpath($element, $search)
+    {
+        $selectorsHandler = $this->session->getSelectorsHandler();
+        $literal = $selectorsHandler->xpathLiteral($search);
+
+        // To be able to work on mink 1.6 (ezplatform) & mink 1.5 (5.4+ezpublish-community) w/o deprecation exceptions
```

Comment:
This doesn't apply anymore, since this is ezplatform@master (6.4 and onward), does it ?